packages/core/contracts/financial-templates/perpetual-multiparty/PerpetualPositionManager.sol

```diff
@@ -512,6 +523,8 @@ contract PerpetualPositionManager is FeePayer, AdministrateeInterface {
      * @dev This burns all tokens from the caller of `tokenCurrency` and sends back the resolved settlement value of
      * `collateralCurrency`. Might not redeem the full proportional amount of collateral in order to account for
      * precision loss. This contract must be approved to spend `tokenCurrency` at least up to the caller's full balance.
+     * @dev Note that this function does not call the updateFundingRate modifier to update the funding rate as this
+     * function is only called after an emergency shutdown & there should be no funding rate updates after the shutdown.
      * @return amountWithdrawn The actual amount of collateral withdrawn.
      */
     function settleEmergencyShutdown()
```

Comment:
So just to clarify, no need to `updateFundingRate()` on this call because all collateral and debt should be frozen at this point?

Reply:
correct